docs/docs/v2v-migration-guide.md

```diff
@@ -0,0 +1,281 @@
+    ![The Task screens shows the nbdinfo install task, with a status indicator](./assets/nbdinfo_progress_2.png)
+    Thes task will be removed from the list upon completion.
+
+4. Once the installation finishes, the `nbdinfo` install screen updates automatically. It lets you know if everything was completed successfully and flags any warnings that came up during the process:
+    ![The Import screen showing a completed nbdinfo install, with warnings](./assets/nbdinfo_install_complete.png)
+
+
+#### Installing the other dependencies
+
+:::warning
+You can compile `nbdkit` and `libnbd` from source on GitLab, but the correct dependencies must be in place for `nbdinfo` and VDDK.
+:::
+
+##### Installing VDDK
+
+1. Download the VMware Virtual Disk Development Kit (VDDK) from the [Broadcom developer portal](https://developer.broadcom.com/sdks/vmware-virtual-disk-development-kit-vddk/9.0?ref=xen-orchestra.com). 
+2. Select the *tar.gz* archive, then drag and drop it directly into the Xen Orchestra interface:
+![Import screen showing a VDDK import dialog](./assets/upload_vddk.png)
+An **install button** will appear.
+3. Click the button: 
+![Button to confirm the installation of an uploaded VDDK archive](./assets/install_vddk_button.png)
+The installation should take just a few seconds.
+Once the installation is done, a **transfer form** will appear.:
+![Form for transfering a VM from VMware to Xen Orchestra](./assets/vddk_transfer_form.png)
+:::warning
+Filling in this form and clicking the **Connect** button will start the VM import. However, make sure your VMware environment is ready! 
+
+Read [Preparing the VMware Environment](#-preparing-the-vmware-environment) to know more.
+:::
+
+## 👨‍🍳 Preparing the VMware Environment
+
+### Prerequisites
+
+Before starting the migration, make sure your VMware environment meets the following conditions:
+
+#### Network
+XO must be able to connect to the vsphere/esxi through the port running the web UI (default port: 443) and vddk (default port: 902). At the time of writing, there is no solution to select one network or another. If possible, keep one single network path from XO to VMware.
+
+#### VMware disk support
+
+##### ✅Supported 
+
+- **Native snapshots** are supported. To migrate these:
+    1. Shut them down completely.
+    2. Remove all snapshots before attempting the migration.
+- **Warm migration** is supported. If warm migration fails:
+        1. Stop the VM.
+        2. Remove any snapshots.
+
+##### ⚠️ Unofficial support
+
+VSAN configurations have not been fully tested, but they are expected to work.
+
+##### ❌ Unsupported
+
+The following VMware disk types are not supporetd: 
+
+- RDM (Raw Device Mapping)
+- Independent disks
+- Physical compatibility mode RDMs
+- Encrypted virtual disks (unless you have the correct credentials)
+
+:::warning
+Starting with Xen Orchestra 5.110, NFS-based migration and VSAN exports are no longer available.
+:::
+
+### Step-by-step procedure 
```

Comment:
this is the" sure fire way", with a lot of manual steps, but the ability to detect and correct issues without impactg production

there is also a fire and forget way " launch the migration with `stop source`" enabled , and then start the VM when the migration is done. The user won't know when the source VM will be stopped.  
This is advised only for less critical VM, and after he firsts migrations have gone well